.travis.yml

```diff
@@ -3,30 +3,46 @@ services:
 - docker
 install:
 - pip install twine wheel pyOpenSSL
-script:
-- |
+- pip install -r requirements.txt
 
-  export last_commit_author=`git log -1 --pretty=format:%ae`
+branches:
```

Comment:
do we need this? we want it to run on all branches but only commit where it is not master - which I think you have covered in the check later

Reply:
Removed.